### ISSUE TITLE
Producer property warnings on consumer creation

### ISSUE THREAD

**Issue by apeloquin-agilysys** [confluentinc/confluent-kafka-javascript#40]
```ts
const kafka = new Kafka({
  kafkaJS: {
    brokers: ["localhost:9092"]
  }
});
const consumer = kafka.consumer({
  kafkaJS: {groupId: GROUP_ID}
});
await consumer.connect();
```

Results in the following log entries, even though I have clearly not specified `retry.backoff.ms` or `retry.backoff.max.ms`.

```
{
  message: '[thrd:app]: Configuration property retry.backoff.ms is a producer property and will be ignored by this consumer instance',
  fac: 'CONFWARN',
  timestamp: 1715289786815
}
{
  message: '[thrd:app]: Configuration property retry.backoff.max.ms is a producer property and will be ignored by this consumer instance',
  fac: 'CONFWARN',
  timestamp: 1715289786816
}
```

**Comment by milindl** [comment]
This will be fixed in the next release, it was an issue with librdkafka. It only affects the logging. 
The properties are specified internally for compatibility with the default KafkaJS configurations.

**Comment by milindl** [comment]
Closing this as I released the v0.1.15-devel
